### PR TITLE
Modified the Terms

### PR DESCRIPTION
**Pull request by David-Chadwick**
This is item iv) from PR#141

### CODE REVIEW

**Comment by msporny** [comment]
Ping again wrt. resolving these conflicts, I can't do anything w/ this PR w/ conflicts other than manually trying to fix them and I might remove some text in that process. Please advise wrt. next steps.

**Comment by David-Chadwick** [comment]
@msporny . Sorry I am still new to this. Do I need to update my PR first to be synced with the latest version?

**Comment by msporny** [comment]
> Sorry I am still new to this. Do I need to update my PR first to be synced with the latest version?

Yes, unfortunately it's going to require you to do what's called a "rebase on top of gh-pages", fix the conflicts, and then push a new version. What might be easier is to just reapply your PR to the latest version of gh-pages. That is, do a fresh checkout and submit another PR. :(

**Comment by davidlehn** [comment]
Please also remove the .DS_Store file from this PR before merging.

File: terms.html [hunk 1]
```diff
@@ -8,14 +8,14 @@
...
+An assertion made about a <a>subject</a>.
   </dd>
   <dt><dfn data-lt="credential|credentials">credential</dfn></dt>
   <dd>
-A set of one or more claims made by the same <a>entity</a> about a
-<a>subject</a>. A <dfn data-lt="verifiable credentials">verifiable credential</dfn>
+A set of one or more claims made by the issuer. A <dfn data-lt="verifiable credentials">verifiable credential</dfn>
 is a credential that is tamper-resistant and whose authorship can be
 cryptographically verified.
+ Note. The claims in a credential may be about different subjects.
```
Comment (davidlehn) [inline terms.html]: Is this supposed to be "Note: "?

File: terms.html [hunk 2]
```diff
@@ -54,8 +54,8 @@
   </dd>
   <dt><dfn data-lt="holders|holder's">holder</dfn></dt>
   <dd>
-An <a>entity</a> that is in control of one or more <a>credentials</a>.
-A holder is typically also the primary <a>subject</a> of the
+An <a>entity</a> that is in control of one or more <a> verifiable credentials</a>.
```
Comment (davidlehn) [inline terms.html]: Remove space before verifiable.

File: terms.html (hunk 1 above)
Comment (David-Chadwick) [reply terms.html]: Yes

**Comment by David-Chadwick** [comment]
@msporny 
"For future reference, if you created your branches directly on the vc-data-model repository, I would've been able to fix the conflicts and clean up the PR"
Unfortunately I dont have write permission to w3c folder so I have to fork to my own repository.

**Comment by msporny** [comment]
Replaced by #167, which has been merged, closing.

**Pull request closed without merging**